workflow/kfdrc-jointgenotyping-refinement-workflow.cwl

```diff
@@ -0,0 +1,321 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-jointgenotyping-refinement-workflow
+label: Kids First DRC Joint Genotyping Workflow
+doc: |
+  # Kids First DRC Joint Genotyping Workflow
+
+  ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+  Kids First Data Resource Center Joint Genotyping Workflow (cram-to-deNovoGVCF). Cohort sample variant calling and genotype refinement.
```

Comment:
maybe move this one-liner right after the main header.  It sounds silly, but if there isn't enough text before the logo reference, the preview in the cavatica public app page shows the image reference, which is awkward looking. It's purely aesthetic, but a small thing.

Reply:
Eyesore shuffled into obscurity.